### PR TITLE
fix: added error message for missing CVEs

### PR DESCRIPTION
**Pull request by aokirisaki**
Signed-off-by: Ana-Roberta Lisca <ana.kagome@yahoo.com>

**What type of PR is this?**

**Which issue does this PR fix**:
#803 

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: errors/errors.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ var (
 	ErrCliBadConfig            = errors.New("cli: bad config")
 	ErrRepoNotFound            = errors.New("repository: not found")
 	ErrRepoIsNotDir            = errors.New("repository: not a directory")
+	ErrCVENotFound             = errors.New("CVE: no CVE found for image")
```
Comment (andaaron) [inline errors/errors.go:10]: We shouldn't have such an error. Having no CVEs is a good thing.
And I also think this should be handled on ZLI side. It's a matter of how we present the results to the user.

**Review by andaaron — CHANGES_REQUESTED** [review]

**Comment by rchincha** [comment]
rebase and update linting

**Review by andaaron — COMMENTED** [review]
Please add a comment with an example of the command, and command output.

**Comment by aokirisaki** [comment]
> Please add a comment with an example of the command, and command output.

Here is an example output:
```
$ bin/zli-linux-amd64 cve config-multiple-cve5000 --image busybox:latest
No CVEs found for image
```

**Review by andaaron — APPROVED** [review]

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**